documentation/RLC-customization-doc.md

```diff
@@ -0,0 +1,167 @@
+
+Below is the example code to call the helper.
+
+```typescript
+import MetricsAdvisor, { paginate } from "@azure-rest/ai-metricsadvisor";
+import { DefaultAzureCredential } from "@azure/identity";
+
+const client = MetricsAdvisor("https://<endopoint>", new DefaultAzureCredential());
+
+const initResponse = await client.listDataFeeds({
+  queryParameters: {
+    dataFeedName: "js-test-",
+    $skip: 1,
+    $maxpagesize: 1,
+  },
+});
+
+const dataFeeds = paginate(client, initResponse);
+for await (const dataFeed of dataFeeds) {
+  console.log(data);
+}
+```
+
+## Custom data transform helpers
+
+There may be times in which transforming the data from the service would be beneficial. When a transformation is common for our customers we may decide to expose helper transformation functions. These helper transformations are optional and customers can decide to use them or not, the calls maintain the original data form from the Service.
+
+If we export `toDataFeedDetailResponse` which may convert the REST model to a common one, so that the customers could call this way:
+
+```typescript
+import MetricsAdvisor, { toDataFeedDetailResponse } from "@azure-rest/ai-metricsadvisor";
+import { DefaultAzureCredential } from "@azure/identity";
+
+const client = MetricsAdvisor("https://<endpoint>", new DefaultAzureCredential());
+const listResponse = await client.listDataFeeds(<parameter>);
+if (listResponse.status != "201") {
+  throw new Error("Error");
+}
+
+// Transforms service data into a more useful shape
+const formattedDatafeed = toDataFeedDetailResponse(listResponse);
+```
+
+## Multi-client packages
+
+There are cases where 2 services are closely related that most users will need to use both in the same application, in this case, we may opt for multi-client packages. Each client can be imported individually without a top-level client, this is to work nicely with bundler TreeShaking.
+
+We could leverage the autorest batch option and enable multi-client flag in our `README.md` to generate two or more service clients.
+
+Here is an example in metrics advisor, we have two clients `MetricsAdvisorClient` and `MetricsAdvisorAdministrationClient`.
+
+### Use multi-client flag and batch option
+
+Add the `multi-client` flag in our readme and use the `batch` autorest option to create the two clients:
+
+```yaml $(multi-client)
+batch:
+  - metrics-advisor: true
+  - metrics-advisor-admin: true
+```
+
+### Specify configurations for each individual clients
+
+For each individual clients specify your client name and swagger file. Make sure that you don't have one Swagger with operations that are designed to be in two different clients so that clients should correspond to a clear set of Swagger files.
+
+```yaml $(metrics-advisor) == true
+title: MetricsAdvisorClient
+description: Metrics Advisor Client
+output-folder: ${project_folder}/src
```

Comment:
where is this project folder being defined ?

Reply:
updated